bots/Bots.md

```diff
@@ -19,13 +20,15 @@
   - [iam\_user\_force\_password\_change](#iam_user_force_password_change)
   - [igw\_delete](#igw_delete)
   - [kms\_enable\_rotation](#kms_enable_rotation)
+  - [lambda\_with\_admin\_privileges](#lambda_with_admin_privileges)
   - [mark\_for\_stop\_ec2\_resource](#mark_for_stop_ec2_resource)
   - [rds\_quarantine\_instance](#rds_quarantine_instance)
   - [s3\_delete\_acls](#s3_delete_acls)
   - [s3\_delete\_permissions](#s3_delete_permissions)
   - [s3\_enable\_encryption](#s3_enable_encryption)
   - [s3\_enable\_logging](#s3_enable_logging)
   - [s3\_enable\_versioning](#s3_enable_versioning)
+  - [s3\_enable\_versioning](#s3_only_allow_ssl)
```

Comment:
copy paste mistake, not enable versioning

Reply:
fixed